### ISSUE TITLE
Questions about the checkpoint of NR3D 

### ISSUE THREAD

**Issue by Limx1630** [dfki-av/MiKASA-3DVG#6]
Thank you for your great work, but I have some questions regarding this. First, when I load the NR3D pre-trained weights into our model, it indicates that some parameters do not match, as shown in the error message. However, I did not encounter this issue with the SR3D weights. I greatly appreciate your explanation.
![微信图片_20241212180216](https://github.com/user-attachments/assets/3210fc47-f642-49a6-bb98-43305577ff7e)

**Comment by birdy666** [comment]
Thanks for reaching out! This is a known bug, please pull the new code or check the modification in this [commit](https://github.com/dfki-av/MiKASA-3DVG/commit/74cdffd8c182d4afa012ceeb8a5b35f298161ccb)
Thanks a lot.

**Comment by Limx1630** [comment]
Thank you for your explanation. After that, I successfully tested the NR3D and SR3D datasets using the checkpoint you provided. However, the results show some difference compared to the accuracy reported in your paper. For example, the accuracy for NR3D is 62.68% (vs. 64.44% in your paper) and the accuracy for SR3D is 73.82% (vs. 75.2% in your paper). Could you please help me understand the possible reasons for this? Is there any mistake in my process?

**Comment by birdy666** [comment]
Could you please share the accuracy of all the category? Thanks

**Comment by Limx1630** [comment]
nr3d : hard & 57.452830 & 0.339965 \\easy & 68.111202 & 0.342052 \\v-dep & 63.656174 & 0.361623 \\v-indep & 62.192930 & 0.270546 \\all & 62.677355 & 0.286790 \\among-true & 70.909820 & 0.204936 \\
sr3d:hard & 65.411631 & 0.188065 \\easy & 77.399839 & 0.319045 \\v-dep & 70.691489 & 0.701140 \\v-indep & 73.956640 & 0.248543 \\ all & 73.818120 & 0.218310 \\

**Comment by birdy666** [comment]
Thank you very much for reporting this! I found the problem. Please find the new [commit](https://github.com/dfki-av/MiKASA-3DVG/commit/3fb9ca072a81dfa6a097980b0dd4e0f4cd01cee5) for the update.
(note that this change will not affect the performance much if you train the model from scratch)
Thanks again.

**Comment by Limx1630** [comment]
Thank you for your response.  the accuracy has improved to some extent after modified the code , but there is still a gap compared to the results reported in the paper. Below are the metrics I obtained on the NR3D and SR3D datasets:

NR3D:
hard: 58.96
easy: 68.38
v-dep: 64.57
v-indep: 63.08
all: 63.58
SR3D:

hard: 67.44
easy: 77.47
v-dep: 68.91
v-indep: 74.72
all: 74.48

Could there be any key steps I missed during the reproduction process, or other potential errors I should check for?

**Comment by birdy666** [comment]
hmmm then I have no idea I am sorry :( I cloned it from github again and was able to reproduce it. I will check if others have the same problem. Will let you know if there is any update.
